docs/src/developer_guide.md

```diff
@@ -293,7 +338,7 @@ like the following:
 ```julia
 import ClimaDiagnostics: average_pre_output_hook!, HDF5Writer, NetCDFWriter, ScheduledDiagnostic
 
-function parse_yaml(parsed_args, target_space)
+function parse_yaml(parsed_args, source_space)
```

Comment:
src/netcdf_writer_coordinates.jl still has a function `target_coordinates`, which maybe should be changed to `source_coordinates` now? Also, the docstring for that function doesn't match the function - it has an extra `!` and type annotation

Reply:
It is `target_coordinates` because these are the coordinates in the target Cartesian space